mifospay/src/main/java/org/mifos/mobilewallet/mifospay/savedcards/ui/CardsFragment.java

```diff
@@ -154,6 +154,25 @@ public void onRefresh() {
         });
     }
 
+    private void deleteCard (final int position) {
```

Comment:
instead of passing position in the `deleteCard()` method, pass the cardId itself `mCardsAdapter.getCards().get(position).getId()`

This allows you to make this method reusable and you can call this from other places instead of tying it to adapter position